### PR TITLE
Add [Main] to the list of valid groups for HID.

### PR DESCRIPTION
**Pull request by ywwg**
Fixes issues with calls to resolveGroup returning undefined

### CODE REVIEW

File: res/controllers/common-hid-packet-parser.js [hunk 1]
```diff
@@ -1252,6 +1252,7 @@ class HIDController {
             "[Sampler6]",
             "[Sampler7]",
             "[Sampler8]",
+            "[Main]",
```
Comment (Swiftb0y) [inline res/controllers/common-hid-packet-parser.js:1255]: what group is this again? we also need to add `[App]` IIRC.

Comment (ywwg) [reply res/controllers/common-hid-packet-parser.js:1255]: There seem to have been recent changes that moved, say, vu_meter_l to the "[Main]" group. This list needs to be kept up to date or HID controllers stop working.

Comment (ywwg) [reply res/controllers/common-hid-packet-parser.js:1256]: Added [App]

**Comment by JoergAtGithub** [comment]
I think we have also more valid control group names:

- [Skin]
- [Sampler9...64]

@Holzhaus Could you have a look, if this list is complete?

**Comment by ywwg** [comment]
I wonder if we can get rid of this list completely.  it's only used in two places:

1: inside resolveGroup to decide if the group being resolved is already known.  But all this function does is resolve "deck1" and "deck2" to the appropriate channel.  As written, otherwise it returns the group as-is.  So we could just replace the whole function with a test for deck1 and deck2.  There is an indirect debugging effect that it returns undefined if the group is unknown, but in practice any typoed group name will be caught elsewhere with a missed engine connection or otherwise.

2: inside getActiveFieldGroup, where it is essentially doing the exact same thing -- calling resolvegroup.


So I am going to get rid of this list and see what happens

**Comment by JoergAtGithub** [comment]
Code LGTM! Did vou check that no mapping uses this public list directly?

**Comment by daschuer** [comment]
This seems not the case. LGTM.

**Review by daschuer — APPROVED** [review]

**Pull request merged**